services/spiget/spiget-tested-versions.service.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const { BaseSpigetService, documentation } = require('./spiget-base')
+
+module.exports = class SpigetTestedVersions extends BaseSpigetService {
+  static get route() {
+    return {
+      base: 'spiget/tested-versions',
+      pattern: ':resourceid',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'tested versions',
+      color: 'blue',
+    }
+  }
+
+  async handle({ resourceid }) {
+    const { testedVersions } = await this.fetch({ resourceid })
+    const { versions } = this.transform({ testedVersions })
+    return this.constructor.render({ versions })
+  }
+
+  transform({ testedVersions }) {
+    const earliest = testedVersions[0]
+    const latest = testedVersions.slice(-1)[0]
+    let versions = ''
+    if (earliest === latest) {
+      versions = earliest
+    } else {
+      versions = `${earliest}-${latest}`
+    }
+    return { versions }
```

Comment:
We should add some additional test cases (using mock responses) to validate all of this logic